standard/standard_2.php

```diff
@@ -931,10 +931,10 @@ function proc_nice(int $priority): bool {}
  * @param int $min [optional]
  * @param int $max [optional]
  * @return int A pseudo random value between min
- * (or 0) and max (or getrandmax, inclusive).
+ *  and max (or getrandmax, inclusive).
  */
 #[Pure(true)]
-function rand(int $min = 0, int $max): int {}
```

Comment:
I'm sorry, but according to reflection function has 2 parameters and both are optional. Please see https://3v4l.org/WuoDD . Also according to docs https://www.php.net/manual/en/function.rand.php `If called without the optional min, max arguments rand() returns a pseudo-random integer between 0 and getrandmax()` Could you please elaborate on this?

Reply:
The parameters are both optional, until one parameter is used, then both parameters are needed. 🤔

Reply:
Thank you @voku! Unfortunately, it's not possible at the moment to declare such logic in stubs. I've created a feature request https://youtrack.jetbrains.com/issue/WI-61718 to be able to process such cases.